README.md

```diff
@@ -503,34 +221,25 @@ Accepted log types:
 	1. To test CLI changes locally you can run `node <project_dir>/src/bin/medic-conf.js`. This will run as if you installed via npm.
 	1. To test changes that are imported in code run `npm install <project_dir>` to use the local version of medic-conf.
 
-## compress images
-
-To compress PNGs and SVGs in the current directory and its subdirectories, two commands are available:
-
-	compress-pngs
-	compress-svgs
-
-## Wishlist
-
-* only upload things which have changed (this could be a separate mode - e.g. `update` vs `configure`)
-
 # Releasing
 
-## Do I need to update `release-notes.md`?
```

Comment:
I disagree with the removable of this split and the subsequent re-ordering. However, I would say that because I wrote it.

Reasons I did it this way:
 - These are the two actions you need to perform ("Do I need to update the notes?", "Perform the release")
 - This is the order you do them in
 - Less likely that people's brains won't entirely skip over the release notes points (people naturally skim, titles are there to draw eye to important points)

Not going to block the merge because it's all opinion, but if my points about resonate with you feel free to revert

Reply:
Makes sense, I'll reorder so that it's chronological. The intention of the change was more about the isolation and tone of the header since it didn't match that of the rest of the readme.